crates/core/tedge_watchdog/src/error.rs

```diff
@@ -1,17 +1,24 @@
-use time::error::Parse;
-
+#[cfg(target_os = "linux")]
 use mqtt_channel::MqttError;
-
+#[cfg(target_os = "linux")]
 use tedge_config::CertificateError;
+#[cfg(target_os = "linux")]
 use tedge_config::ConfigSettingError;
+#[cfg(target_os = "linux")]
 use tedge_config::TEdgeConfigError;
+#[cfg(target_os = "linux")]
+use time::error::Parse;
```

Comment:
Instead of adding a target to all these lines, I would keep the error module linux specific and move the non-unix case in `dummy_watchdog.rs`.

Reply:
Thanks for pushing the fix for this.